README.md

```diff
@@ -50,6 +50,38 @@ Android) only, try:
 ./scripts/build-glean-metadata fenix
 ```
 
+For the search service _only_ (see below), the Glean Dictionary also indexes
+legacy Firefox telemetry metadata. To build this index (not necessary for most
+testing), you need to run a seperate script:
+
+```bash
+./scripts/build-legacy-metadata
+```
```

Comment:
I would prefer not to have two scripts here, but it seems like the cleanest approach for now until we can further refactor the code in `__main__.py`  to be a bit modular. Will address in a followup.